VAMobile/src/components/WaygateWrapper.tsx

```diff
@@ -35,7 +43,7 @@ export const WaygateWrapper: FC<WaygateWrapperProps> = ({ children, waygateName
   }
 
   const waygate = waygateEnabled(waygateScreen as WaygateToggleType)
-  if (waygate.enabled === false) {
+  if (waygate.enabled === false && waygateTypeCheck(waygate.type) && (waygate.errorMsgTitle || waygate.errorMsgBody)) {
```

Comment:
Consider throwing an error if the waygate is incorrectly configured, instead of silently ignoring the error